### ISSUE TITLE
What is the alternative for the removal of Cart::prepareDataForOrder()

### ISSUE THREAD

**Issue by MZT7** [bagisto/bagisto#9699]
Once you try to use payPal or paystack for Payment it throws this error after payment is successful:
![Screenshot (97)](https://github.com/bagisto/bagisto/assets/40639877/5719a99b-985c-4261-a3a7-d3c2c2b35a3c)

i found out it was deprecated but what is the alternative to populate the $this->orderRespository->create()

**Comment by amit-webkul** [comment]
Hello, @MZT7 

I just wanted to let you know about our new upgrade guide. It's got all the important updates you need to know, including any methods we've stopped using and what to do instead.

Please check it out when you have a moment! If you have any questions, just give us a shout.

Thanks!

**Comment by MZT7** [comment]
> Hello, @MZT7
> 
> I just wanted to let you know about our new upgrade guide. It's got all the important updates you need to know, including any methods we've stopped using and what to do instead.
> 
> Please check it out when you have a moment! If you have any questions, just give us a shout.
> 
> Thanks!

I have read the guide but what I mean is that when trying to use the package "PayPal" in Packages/webkul it throws the error above because of the missing function in Cart class

 ```
public function success()
    {
        $order = $this->orderRepository->create(Cart::prepareDataForOrder());
// The line above is what causes the error

        Cart::deActivateCart();

        session()->flash('order', $order);

        return redirect()->route('shop.checkout.onepage.success');
    }
```
So I need to know if there is a new function to replace Cart::prepareDataForOrder() so a new ORDER can be created when a user makes a successful payment

**Comment by amit-webkul** [comment]
@MZT7 ,

I wanted to inform you that the method `prepareDataForOrder()` causing issues in the PayPal payment section has been deprecated temporarily. Our team is actively working on resolving this issue.

We expect to have a fix soon and will update you accordingly.

Thank you for your patience.

**Comment by jitendra-webkul** [comment]
Fixed in #9711
